dataprovider/providers/cloud/data_provider.go

```diff
@@ -32,25 +32,19 @@ const (
 	cloudAccountNameField = "cloud.account.name"
 	cloudProviderField    = "cloud.provider"
 	cloudRegionField      = "cloud.region"
-	cloudProjectIdField   = "cloud.project.id"
```

Comment:
From the ECS docs, I think there will be a clear expectation from users that `cloud.project.id` should be populated. WDYT about populating both `cloud.project.*` and `cloud.account.*` with the same data?

Reply:
I proposed this in our daily, but @oren-zohar believes it might be redundant.

Reply:
@uri-weisman @oren-zohar let's go with just populating the cloud.account.* fields for now. The purpose of ECS is to normalize heterogeneous data sources. In this case, by having both project and account, it's doing exactly the opposite. I will work with someone on the ECS team to understand why this is and get back to y'all 👍🏾